### PR TITLE
Translation language code fix

### PR DESCRIPTION
**Pull request by mihirapatel**
Fixed the language code so the voice stays in US English and translation works

### CODE REVIEW

**Comment by dibenede** [comment]
Just had to do a double take. It looks like your PR includes the currency stuff :)

File: portfolio/src/main/java/com/google/sps/utils/AgentUtils.java [hunk 1]
```diff
@@ -107,6 +116,7 @@ public static String getUserInput() {
     } catch (Exception e) {
```
Comment (dibenede) [inline portfolio/src/main/java/com/google/sps/utils/AgentUtils.java:116]: can a more specific type be used here than Exception?

File: portfolio/src/main/java/com/google/sps/utils/AgentUtils.java [hunk 2]
```diff
@@ -37,6 +37,7 @@ public static Output getOutput(QueryResult queryResult, String languageCode) {
       display = object.getDisplay();
       redirect = object.getRedirect();
     } catch (Exception e) {
```
Comment (dibenede) [inline portfolio/src/main/java/com/google/sps/utils/AgentUtils.java:39]: ditto: anything more specific than exception?

File: portfolio/src/main/java/com/google/sps/utils/AgentUtils.java (hunk 1 above)
Comment (mihirapatel) [reply portfolio/src/main/java/com/google/sps/utils/AgentUtils.java:116]: I think Abby is working in the Exceptions for this file, so it should be in her PR

Comment (dibenede) [reply portfolio/src/main/java/com/google/sps/utils/AgentUtils.java:116]: sgtm

**Pull request merged**